app/lib/search/backend.dart

```diff
@@ -90,11 +95,50 @@ class SearchBackend {
         readme: compactReadme(pv.readmeContent),
         health: analysisView.health,
         popularity: mockScores[pv.package] ?? 0.0,
+        maintenance: maintenance,
         timestamp: new DateTime.now().toUtc(),
       );
     }
     return results;
   }
+
+  double _calculateMaintenance(Package p, PackageVersion pv) {
+    final DateTime now = new DateTime.now();
+    final Duration age = now.difference(pv.created);
+    if (age > _twoYears) {
+      return 0.0;
+    }
+
+    double score = 1.0;
+
+    // over a year old?
```

Comment:
Personally I think comments which just say the same thing as the source are not really necessary. Here the line below says `age > _year` which is the same as `over a year old`.